parse.go

```diff
@@ -192,6 +192,9 @@ func (p *Parser) Next() (*Element, error) {
 
 	if elem.Tag == tag.SpecificCharacterSet {
 		encodingNames := MustGetStrings(elem.Value)
+		for i := range encodingNames {
```

Comment:
The pydicom library is considered a good library, strong community. 
Pls update the code so it looks like the python library.
https://github.com/pydicom/pydicom/blob/d5f9ae9223f4c65a479967e76d6ae5abecefd345/pydicom/charset.py#L721


```python
import pydicom
>>> pydicom.charset.convert_encodings('ISO_2022_IR_6')
/opt/homebrew/lib/python3.11/site-packages/pydicom/charset.py:743: UserWarning: Incorrect value for Specific Character Set 'ISO_2022_IR_6' - assuming 'ISO 2022 IR 6'
  _warn_about_invalid_encoding(encoding, patched)
['iso8859']
```